src/main/proto/ess.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+option java_multiple_files = true;
+package org.ess;
+
+service EncryptedStorageService {
+  rpc get(GetRequest) returns (GetResponse);
+  rpc put(PutRequest) returns (PutResponse);
+  //TODO: Add getKeysSummary (Gives version numbers for all keys in a store) and list by prefix API's
+}
+
+message GetRequest {
+  /*
+    All API's operate within a single storeId.
+    StoreId is keyspace identifier. Ref: https://en.wikipedia.org/wiki/Keyspace_(distributed_data_store)
+    It is up to clients to use single or multiple store's for their usage.
+   */
+  string storeId = 1;
+  string key = 2;
+}
+message GetResponse {
+  bool success = 1;
```

Comment:
I don't think we need to build errors into the messages. This should be done at the gRPC level.

https://cloud.google.com/apis/design/errors
https://cloud.google.com/apis/design/errors#grpc_mapping

Reply:
Yes that is one way to do it but grpc errors are quite limiting. 
Hence there is another way to do it which is also general practice.

For e.g. reason in errorInfo struct is a "string" due to which it is difficult to communicate possible error scenarios to client using enum.
And in most cases clients need to take different actions based on type of error returned.

Reply:
`google.rpc.Status` seems quite flexible, actually. Besides an error code and message, you can provide any other data such as an enum.

https://github.com/googleapis/googleapis/blob/f88ca868801b2fb5fc59f69bde46106f0934e062/google/rpc/status.proto#L46

But I'd rather use the standard error codes and only provide additional details if there's a need to differentiate errors with the same code. Seems like `ErrorCode` already maps well to the provided codes?

https://github.com/googleapis/googleapis/blob/master/google/rpc/code.proto

Then the API can be written in terms of domain types rather than "response" types.

Reply:
Removed the status and ErrorResponse from ResponseObjects. 

Now if httpStatus ==200, then valid response content-entity contains valid GetObjectResponse serialized.
Otherwise if httpStatus != 200, then response content-entity contains ErrorResponse serialized.